src/pages/index.js

```diff
@@ -69,29 +56,82 @@ const handleImageClick = (name, link) => {
 };
 
 const handleAddCardSubmit = (data) => {
-  const cardElement = createCard(data);
-  section.addItem(cardElement);
+  cardModalButton.textContent = "Saving...";
+  api
+    .addNewCard(data)
+    .then((result) => {
+      cardModalButton.textContent = "Save";
+      const cardElement = createCard(result);
+      section.addItem(cardElement);
+      cardFormValidator.disableButton();
+    })
+    .catch((err) => {
+      console.error(err);
+    });
 };
 
 const handleEditProfileSubmit = (formValues) => {
-  userInfoInstance.setUserInfo(formValues);
+  profileModalButton.textContent = "Saving...";
+  api.editProfileData(formValues).then((formvalues) => {
+    profileModalButton.textContent = "Save";
+    userInfoInstance.setUserInfo(formValues);
+  });
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere**